src/octree/octree_iterator.rs

```diff
@@ -36,7 +36,7 @@ impl NodeIterator<CubeNodeReader> {
         }
 
         if let Ok(mut data_map) = octree_data_provider.data(id, &["intensity"]) {
-            match data_map.remove("position") {
+            match data_map.remove("intensity") {
```

Comment:
Is that a fix for something that slipped in by accident in a previous PR?

Reply:
Yes, sorry, I wanted to mention it... I can also make a mini PR for this, if you prefer...

Reply:
I don't think it's a problem, I was just curious :)

Reply:
Ok